pyroma/ratings.py

```diff
@@ -266,13 +268,39 @@ def test(self, data):
 
     def message(self):
         if self._major_version_specified:
-            return (
-                "The classifiers should specify what minor versions of "
-                "Python you support as well as what major version."
-            )
+            return "The classifiers should specify what minor versions of "\
+                   "Python you support as well as what major version."
         return "You should specify what Python versions you support."
```

Comment:
```suggestion
        return "The classifiers should specify what Python versions you support."
```

This still needs to be updated.